components/tyk-gateway/values.yaml

```diff
@@ -149,6 +149,12 @@ gateway:
   # replicaCount specifies number of replicas to be created if kind is Deployment.
   replicaCount: 1
 
+  # minAvailable and maxUnavailable configure thresholds for PodDisruptionBudget resource
+  # per PodDisruptionBudget spec they are mutually exclusive, but this chart allows
+  # setting them both, in which case minAvailable will take precedence
+  minAvailable: ""
```

Comment:
Can you put `minAvailable` and `maxUnavailable` under `pdb` or `podDistributionBudget`?

It will be good to have enabled field too

Reply:
I moved it under `pdb`.